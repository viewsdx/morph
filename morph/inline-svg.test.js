const toSvg = require('./inline-svg.js')
const path = require('path')

test('#toSvg', async () => {
  expect(
    await toSvg(path.join(__dirname, 'inline-svg-fixture.svg'))
  ).toMatchSnapshot()
})
