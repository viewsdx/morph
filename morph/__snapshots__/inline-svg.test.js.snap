// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#toSvg 1`] = `
"InlineSvgFixture Svg
alignSelf < auto
marginTop < 0
marginBottom < 0
marginLeft < 0
marginRight < 0
width < 50
height < 50
SvgGroup
fillOpacity .7
SvgCircle
cx 6.5cm
cy 2cm
r 100
translateX 0
translateY 50
fill < red
stroke < #000
strokeWidth .1cm

SvgCircle
cx 6.5cm
cy 2cm
r 100
translateX 70
translateY 150
fill <fill2 #00f
stroke < #000
strokeWidth .1cm

SvgCircle
cx 6.5cm
cy 2cm
r 100
translateX -70
translateY 150
fill <fill3 green
stroke < #000
strokeWidth .1cm

"
`;
